packages/sonata-extra-bundle/Listener/TimeZoneSubscriber.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Draw\Bundle\SonataExtraBundle\Listener;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\RequestEvent;
+use Symfony\Component\Validator\Constraints\NotBlank;
+use Symfony\Component\Validator\Constraints\Timezone as TimezoneConstraint;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
+
+class TimeZoneSubscriber implements EventSubscriberInterface
+{
+    /**
+     * @var string
+     */
+    private $timezone;
+
+    /**
+     * @var ValidatorInterface
+     */
+    private $validator;
+
+    public function __construct(string $timezone, ValidatorInterface $validator)
```

Comment:
time zone is not needed. We just don't override the default one (set via server configuration) if there is no cookie